source/Keyboard.cpp

```diff
@@ -51,11 +97,16 @@ void Keyboard::keyboard_handle_key([[maybe_unused]]struct wl_listener *listener,
 
   bool handled = false;
   uint32_t modifiers = wlr_keyboard_get_modifiers(device->keyboard);
-  if ((modifiers & WLR_MODIFIER_ALT) && event->state == WLR_KEY_PRESSED)
+  keycodes_states.update_state(event, static_cast<uint32_t>(keycode), modifiers, device->keyboard->xkb_state);
+  char name[30] = {0};
```

Comment:
Why 30?

Reply:
This thing is basically not usefull, it is just a way to show the actual name of the key we press. Practical for debugging

Reply:
Ok, leave as is then for the moment.